gulpfile.js

```diff
@@ -1,18 +1,29 @@
-const gulp  = require('gulp');
+const gulp = require('gulp');
 const browserSync = require('browser-sync').create();
+const browserify = require('browserify');
+const source = require('vinyl-source-stream');
 
 // Static server
 gulp.task('browser-sync', () => {
-    browserSync.init({
-        server: {
-            baseDir: './'
-        }
-    });
+  browserSync.init({
+    server: {
+      baseDir: './src/'
+    }
+  });
 });
 
 gulp.task('watch', ['browser-sync'], () => {
-	gulp.watch('index.html', browserSync.reload);
+  gulp.watch('./src/index.html', browserSync.reload);
+  gulp.watch('./src/css/*.css', browserSync.reload);
+  gulp.watch('./src/js/*.js', browserSync.reload);
 });
 
+gulp.task('browserify', () =>
+    browserify('./jasmine/spec/inverted-index-test.js')
+        .bundle()
```

Comment:
Indenting here is 4 spaces should be 2, I think this should not be ignored from your eslint.